ArcGISRuntimeSDKQt_CppSamples/Layers/IdentifyRasterCell/IdentifyRasterCell.cpp

```diff
@@ -54,12 +48,10 @@ QString defaultDataPath()
 {
   QString dataPath;
 
-#ifdef Q_OS_ANDROID
-  dataPath = "/sdcard";
-#elif defined Q_OS_IOS
+#ifdef Q_OS_IOS
   dataPath = QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation);
 #else
-  dataPath = QDir::homePath();
+  dataPath = QStandardPaths::writableLocation(QStandardPaths::HomeLocation);
```

Comment:
If I recall, we did want to keep all the individual logic in each sample so they are all self-contained. On the other hand, a helper would remove a lot of ifdefs, includes and platform boilerplate from many samples.

At this point, the data path is an implementation detail and not a critical part of the ArcGIS Runtime workflow we need to advertise this way. It's worth a conversation.
@ldanzinger what are your thoughts?

Reply:
I'll open a separate issue for the discussion. Nothing required here.